cunumeric/_ufunc/__init__.py

```diff
@@ -12,6 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+from __future__ import annotations
```

Comment:
We don't *need* it in any empty module (which could be an init, or not). However I have a strong preference for these to be pure boilerplate, so that no thinking is required (and no future issues in case something is added to the module in the future)